### PR TITLE
Standardizing managed exports to all return hresult

### PR DESCRIPTION
**Pull request by jkotalik**

### CODE REVIEW

File: src/AspNetCore/Src/managedexports.cxx [hunk 1]
```diff
@@ -26,30 +26,37 @@ register_callbacks(
...
 )
 {
-    return pHttpContext->GetRequest()->GetRawHttpRequest();
+    *pHttpRequest = pHttpContext->GetRequest()->GetRawHttpRequest();
+    return S_OK;
 }
 
 EXTERN_C __MIDL_DECLSPEC_DLLEXPORT
-HTTP_RESPONSE*
+HRESULT
```
Comment (pan-wang) [inline src/AspNetCore/Src/managedexports.cxx:41]: as this function always return s_ok, why not make it void

File: src/AspNetCore/Src/managedexports.cxx [hunk 2]
```diff
@@ -110,17 +117,18 @@ http_set_managed_context(
 }
 
 EXTERN_C __MIDL_DECLSPEC_DLLEXPORT
-VOID
+HRESULT
```
Comment (pan-wang) [inline src/AspNetCore/Src/managedexports.cxx:120]: what's the motivation for this change?

Comment (jkotalik) [reply src/AspNetCore/Src/managedexports.cxx:120]: I wanted to standardize all pInvokes to return HRESULTs. In the future, if the implementation of the functions changes and suddenly we can return an invalid HRESULT, I don't want to have to make a second method.

**Comment by jkotalik** [comment]
Closing for now as this may be unnecessary.
